README.md

```diff
@@ -111,6 +122,10 @@ CubiCapture view only works when the view is locked to portrait orientation (eve
 
 ## CubiCapture Features
 
+### Relocating After Loss of Tracking
+
+In cases where the scan is interrupted due to loss of tracking or the app going to the background, the SDK will now attempt to relocate and continue the scan. This is achieved by prompting the user to return to a previously scanned area, so that the SDK can reestablish tracking.
+
 ### Scene Reconstruction
 
```

Comment:
Add the missing event messages and codes to **The event codes** section

Reply:
Done.

Reply:
SDK 2.3.1 had these in release notes 
- New status codes (code 3, 5, 18, 27, 29, 30 and 40-49 and 57) 
so I think it would be proper to have the new events also added to release notes in 2.4.1

Reply:
Also done.

Reply:
@therauli How about relocation events?

Reply:
Hmm, there were already entries for events 80-83 in the `README.md` and the Google doc, but for 80-82 those texts don't match what's in the code.